src/mito2/src/region/opener.rs

```diff
@@ -713,8 +728,8 @@ where
 
     let series_count = version_control.current().series_count();
     info!(
-        "Replay WAL for region: {}, rows recovered: {}, last entry id: {}, total timeseries replayed: {}, elapsed: {:?}",
-        region_id, rows_replayed, last_entry_id, series_count, now.elapsed()
+        "Replay WAL for region: {}, provider: {:?}, rows recovered: {}, last entry id: {}, total timeseries replayed: {}, elapsed: {:?}",
+        region_id, provider, rows_replayed, last_entry_id, series_count, now.elapsed()
     );
```

Comment:
Do we have a log for the topic_latest_entry_id?

Reply:
The outside will log the replay from entry id (which is max(flush entry id, topic latest entry id)) and the flush entry id.